Directory.Build.props

```diff
@@ -6,14 +6,14 @@
   <PropertyGroup>
     <!-- Airship native SDK versions -->
     <AirshipAndroidVersion>17.5.0</AirshipAndroidVersion>
-    <AirshipAndroidNugetVersion>17.5.0</AirshipAndroidNugetVersion>
+    <AirshipAndroidNugetVersion>17.5.0.1</AirshipAndroidNugetVersion>
```

Comment:
looks weird

Reply:
yeah, I don't particularly care for the versioning scheme, but this is how a lot of xamarin/maui binding libs are versioned... we can usually avoid it, but in this case, we don't have a newer Android SDK release to bump to. 🤷